### PR TITLE
Install rustc

### PR DESCRIPTION
**Pull request by shyouhei**
We are not going to `apt-get install rustc`, because what is installed using apt-get tends to be too old, compared to what we need (1.60.0+).

### CODE REVIEW

**Comment by shyouhei** [comment]
This pull request is needed for https://github.com/ruby/ruby/pull/5870

File: Dockerfile [hunk 1]
```diff
@@ -35,11 +52,15 @@ COPY --from=assets /etc/ssl /etc/ssl
...
 COPY --from=assets /etc/sudoers.d /etc/sudoers.d
+COPY --from=assets /rust /rust
+ENV RUSTUP_HOME=/rust/rustup
+ENV PATH=${PATH}:/rust/cargo/bin
 
 RUN set -ex                                           \
  && apt-get update                                    \
  && apt-get install ${packages}                       \
     libjemalloc-dev openssl ruby tzdata valgrind sudo \
+    libcapstone-dev                                   \
```
Comment (XrXr) [inline Dockerfile:63]:
```suggestion
```
It should work without this since the Rust crate we use takes care of building `libcapstone` from source.

Comment (shyouhei) [reply Dockerfile:63]: This is for CI.  If precompiled library in our docker image reduces build time, I think it's worth.

**Comment by shyouhei** [comment]
No plans to merge this for now.  Will revisit later if needed again.

**Comment by shyouhei** [comment]
Bump! We now in fact need this. Let us revive.

**Comment by shyouhei** [comment]
>  Is there some way to test this before merging?

Actually this repo's CI is not working as expected.  It already pushed images based on this pull request.  So yeah, [we found a problem already](https://github.com/ruby/ruby/pull/8028), and was fixed.  Thanks for approval.

**Pull request merged**